src/only-virtual/OnlyVirtualPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class OnlyVirtualPage extends React.Component {
+    render() {
+        return <div>This site currently in Beta version, only Virtual Accounts are allowed.</div>;
```

Comment:
This is definitely not good enough, I submitted so you can review, and not sure if we want to merge into master, or only merge to a release branch.